examples/jsm/controls/TransformControls.js

```diff
@@ -75,6 +75,28 @@ var TransformControls = function ( camera, domElement ) {
 
 	var ray = new Raycaster();
 
+	var intersectObjectsWithRay = function( object, ray, includeInvisible ) {
+
+		var allIntersections = ray.intersectObjects( [ object ], true );
```

Comment:
If there is just a single object, use `intersectObject()`. This avoids to create an array instance inline every time `intersectObjectsWithRay()` is called.

Sidennote: `ray` is actually an instance of `Raycaster` not `Ray`. I think I would rename the variable to `raycaster` for clarity reasons.

Reply:
Good call! Renamed variable to `raycaster` and switched to `intersectObject` instead.